com.unity.visualeffectgraph/Editor/Inspector/VisualEffectEditor.cs

```diff
@@ -1208,11 +1208,11 @@ public RendererEditor(params VFXRenderer[] renderers)
             public static readonly string[] s_DefaultRenderingLayerNames = GetDefaultRenderingLayerNames();
             private static string[] GetDefaultRenderingLayerNames()
             {
-                //Find UnityEditor.RendererEditorBase.defaultRenderingLayerNames by reflection to avoid any breakage due to an API change
+                //Find UnityEditor.RendererEditorBase.defaultPrefixedRenderingLayerNames by reflection to avoid any breakage due to an API change
                 var type = Type.GetType("UnityEditor.RendererEditorBase, UnityEditor");
                 if (type != null)
                 {
-                    var property = type.GetProperty("defaultRenderingLayerNames", BindingFlags.Static | BindingFlags.GetField | BindingFlags.NonPublic | BindingFlags.Public);
+                    var property = type.GetProperty("defaultPrefixedRenderingLayerNames", BindingFlags.Static | BindingFlags.GetField | BindingFlags.NonPublic | BindingFlags.Public);
```

Comment:
This is one of the reason it requires a trunk change, right ?
Was it already correctly implemented without the C++ change ? (aka, no exception)

Reply:
yes exactly. it was working before but the behavior was inconsistent with what SRPs offered, So i updated to get the same behavior for builtin.